chains/lisk/test/main.go

```diff
@@ -0,0 +1,43 @@
+package main
```

Comment:
I am moving away from creating a test folder for integration test. 

Can you create a `watcher_integration_test.go` and put this test into that file? The test should have prefix `TestIntegration_...`

After you are done with the test, add `test.Skip()` so that the CI system will skip the test